Tools/TestWebKitAPI/Tests/WebKitCocoa/WritingTools.mm

```diff
@@ -3243,7 +3243,7 @@ InstanceMethodSwizzler swizzler(PAL::getWTWritingToolsClass(), @selector(showToo
         for (NSView *subview in [webView subviews]) {
             if ([subview isKindOfClass:[_WTTextEffectView class]]) {
                 foundEffectView = true;
-                EXPECT_EQ(subview.frame, NSMakeRect(0, 0, 800, 568));
```

Comment:
What's the error on this one?

Reply:
(The failure was something along the lines of "no `operator==` for `NSRect`")

Reply:
oh, this file needs a `#include "TestCocoa.h"`

Reply:
Ah, that works for me too.

(It looks like `EXPECT_TRUE(NSEqualRects(…))` is a pattern elsewhere in this file as well, so we could probably replace those at some point)

Reply:
It would be nice to do that, yes, because `EXPECT_EQ` has a better output. cc @rr-codes if you get around to it at some point!